dashboard/config/upstream-usb.config

```diff
@@ -5037,25 +5042,6 @@ CONFIG_USB_DUMMY_HCD=y
 # end of USB Peripheral Controller
 
 # CONFIG_USB_CONFIGFS is not set
-# CONFIG_USB_ZERO is not set
-# CONFIG_USB_AUDIO is not set
-# CONFIG_USB_ETH is not set
-# CONFIG_USB_G_NCM is not set
-# CONFIG_USB_GADGETFS is not set
-# CONFIG_USB_FUNCTIONFS is not set
-# CONFIG_USB_MASS_STORAGE is not set
-# CONFIG_USB_GADGET_TARGET is not set
-# CONFIG_USB_G_SERIAL is not set
-# CONFIG_USB_MIDI_GADGET is not set
-# CONFIG_USB_G_PRINTER is not set
-# CONFIG_USB_CDC_COMPOSITE is not set
-# CONFIG_USB_G_NOKIA is not set
-# CONFIG_USB_G_ACM_MS is not set
-# CONFIG_USB_G_MULTI is not set
-# CONFIG_USB_G_HID is not set
-# CONFIG_USB_G_DBGP is not set
-# CONFIG_USB_G_WEBCAM is not set
-CONFIG_USB_RAW_GADGET=y
```

Comment:
`CONFIG_USB_RAW_GADGET` should stay. You need to run the script on `usb-fuzzer` branch. Or just revert the changes to the USB config and I'll regenerate it.

Reply:
I will revert it then. Mostly wanted you to look at the changes caused by the script changes.